src/windows/line.rs

```diff
@@ -165,4 +174,33 @@ impl Lines {
             self.lines.sort_by_key(|x| x.rva);
         }
     }
+
+    pub(super) fn retain(&self, rva: u32, len: u32) -> Option<Lines> {
+        // A symbol space can be splited in several chunks
+        // so we need to retain the lines which are in the different chunks
+        if self.lines.is_empty() {
+            return None;
+        }
+        if rva == self.lines.first().unwrap().rva {
+            let last = self.lines.last().unwrap();
+            if last.rva + last.len == rva + len {
+                return None;
+            }
+        }
+        Some(Lines {
+            lines: self
+                .lines
+                .iter()
+                .filter_map(|line| {
+                    if rva <= line.rva && line.rva + line.len <= rva + len {
+                        Some(line.clone())
+                    } else {
+                        None
+                    }
+                })
+                .collect(),
+            is_sorted: true,
+            last_rva: 0,
+        })
```

Comment:
nit: It would be nice to split this out into a function with a descriptive name, such as `findLinesForRange()` or `gatherLinesForRange()`.